app/src/main/java/org/hackillinois/android/database/dao/EventDao.kt

```diff
@@ -10,6 +10,9 @@ interface EventDao {
     @Query("SELECT * FROM events")
     fun getAllEvents(): LiveData<List<Event>>
 
+    @Query("SELECT * FROM events")
+    fun getAllEventsList(): List<Event>
```

Comment:
Is it possible to condense this and the query above? They're so similar.

Reply:
Unfortunately, I don't think so :/

I'm thinking that maybe at some point we should add favorited into the event database, then this would be an entirely different request.